File: FurnitureFront/src/app/models/order.ts
import { Product } from 'src/app/models/product';
import { OrderLine } from 'src/app/models/order-line';
import { Customer } from 'src/app/models/customer';
export interface Orders {
    id: number;
    customerId: number;
    date: Date;
    statusId: number;
    orderLines: OrderLine[];
    customer: Customer;
}

File: FurnitureBackend/Webshop.Domain/Order.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Webshop.Domain
{
    public class Order
    {
        public int Id_O { get; set; }
        public int CustomerId { get; set; }
        public DateTime Date { get; set; }
        public int StatusId { get; set; }        
        public Status Status { get; set; }
        public List<OrderLine> OrderLines { get; set; }
        public Customer Customer { get; set; }
    }
}
